pkg/network/usm/ebpf_ssl.go

```diff
@@ -694,3 +682,34 @@ func deleteDeadPidsInMap(manager *manager.Manager, mapName string, alivePIDs map
 
 	return nil
 }
+
+// deleteDeadPidsInSSLCtxMap finds a map by name and deletes dead processes.
+func deleteDeadPidsInSSLCtxMap(manager *manager.Manager, alivePIDs map[uint32]struct{}) error {
```

Comment:
why not using the MapCleaner interface?

Reply:
As we discussed, due to the way the cleanup is being done we will keep it like that